src/tokenizer.rs

```diff
@@ -649,12 +699,16 @@ fn consume_quoted_string<'a>(tokenizer: &mut Tokenizer<'a>, single_quote: bool)
                 if !tokenizer.is_eof() {
                     match tokenizer.next_byte_unchecked() {
                         // Escaped newline
-                        b'\n' | b'\x0C' => tokenizer.advance(1),
+                        b'\n' | b'\x0C' => {
+                            tokenizer.advance(1);
+                            tokenizer.seen_newline(false);
+                        }
                         b'\r' => {
                             tokenizer.advance(1);
                             if tokenizer.next_byte() == Some(b'\n') {
                                 tokenizer.advance(1);
                             }
+                            tokenizer.seen_newline(false);
```

Comment:
This is slightly confusing, but I guess it's nicer than call seen_newline with true before this... maybe deserves a comment?